deltachat-ffi/deltachat.h

```diff
@@ -5895,6 +5895,70 @@ void dc_event_unref(dc_event_t* event);
 /// `%1$s` will be replaced by human-readable date and time.
 #define DC_STR_DOWNLOAD_AVAILABILITY      100
 
```

Comment:
the little gap is because of two strings that will probably be added by https://github.com/deltachat/deltachat-core-rust/pull/2669 - less refactoring.